viewflow/lock.py

```diff
@@ -1,16 +1,34 @@
 """
-Prevents unconsistent db updates for flow.
+Prevents inconsistent db updates for flow.
 """
 import time
 import random
 from contextlib import contextmanager
 
-from django.core.cache import cache
+from django.core.cache import DEFAULT_CACHE_ALIAS
 from django.db import transaction, DatabaseError
+from django.conf import settings
 
 from viewflow.exceptions import FlowLockFailed
 
 
+CACHE_BACKEND = getattr(settings, 'VIEWFLOW_LOCK_CACHE_BACKEND', DEFAULT_CACHE_ALIAS)
```

Comment:
I get your point, but why didn't you tell me that before merging. I could have created a callable, that has a configurable cache. Which I will do now.